contrib/babelfishpg_tsql/src/pltsql_coerce.c

```diff
@@ -1481,13 +1482,23 @@ tsql_func_select_candidate(List *names,
 }
 
 static bool
-is_tsql_char_type_with_len(Oid type)
+is_tsql_char_type_with_len(Oid type, bool is_case_expr)
 {
+	bool					result = false;
```

Comment:
NIT: No need of setting this explicitly to false.

Reply:
resolved